tests/test_config_files.py

```diff
@@ -66,9 +66,9 @@ def test_MTMount(self):
             config_package_root=self.config_package_root,
         )
 
-    def test_Rotator(self):
+    def test_MTRotator(self):
         self.check_standard_config_files(
-            sal_name="Rotator",
+            sal_name="MTRotator",
             module_name="lsst.ts.rotator",
```

Comment:
Shouldn't the module name also be updated to lsst.ts.mtrotator?

Reply:
Yes. Good catch. (I updated this package before the rename and forgot to go back).

I fixed that and checked that the tests run.